File: Greedy/week2_1_minimum_cost_of_buying_candies_with_discount/szzng.py
# Runtime 42 ms Beats 85.15%
# Memory 13.8 MB Beats 45.57%

from typing import List


class Solution:
    def minimumCost(self, cost: List[int]) -> int:
        cost.sort(reverse=True)
        result = 0
        for i in range(len(cost)):
            if i % 3 == 2:
                continue
            result += cost[i]
        return result

File: 07_02_Greedy/92_1_minimum_cost_of_buying_candies_with_discount/szzng.js
// Runtime 53 ms Beats 93.60%
// Memory 41.6 MB Beats 99.20%

var minimumCost = function(cost) {
    cost.sort((a,b)=>b-a)
    var result = 0
    var costLen = cost.length

    for (var i=0; i<costLen; i++){
        if (i%3===2) continue
        result += cost[i]
    }

    return result
};